### ISSUE TITLE
Hive and Command Station HP should be shown next to the timer

### ISSUE THREAD

**Issue by TheRealNin** [TheRealNin/NS2Combat#14]

**Comment by DalinSeivewright** [comment]
Should this be the objective HP of your own team or are you planning showing the HP of both for both teams?

**Comment by TheRealNin** [comment]
The standalone combat game would show the HP of both command structures next to the timer, but maybe for fairness it should only show your own team's....

**Comment by DalinSeivewright** [comment]
Yeah, I have mixed feelings on both.  On one hand I think that hiding the enemies Tech Point HP would force team communication to happen "Get over there now Hive is at 10%" sort of thing.  On the other hand, that doesn't seem to do much lately anyways.  Just seeing the number low could rouse the troops...  so you could try either way.

Not sure if standalone blinked, but it might be helpful to blink both hp indicators when getting damaged/low damage.  Oh.... maybe even have a team-only indicator for when the tech point is getting healed?

**Comment by GhoulofGSG9** [comment]
Maybe only update the enemy's command stations hp when it got spotted by a teammate. Grey it out when it hasn't spotted for more than 20 secs or so.
